packages/astro/src/core/errors/errors-data.ts

```diff
@@ -1270,7 +1258,7 @@ export const UnsupportedConfigTransformError = {
 	title: 'Unsupported transform in content config.',
 	message: (parseError: string) =>
 		`\`transform()\` functions in your content config must return valid JSON, or data types compatible with the devalue library (including Dates, Maps, and Sets).\nFull error: ${parseError}`,
-	hint: 'See the devalue library for all supported types: https://github.com/rich-harris/devalue',
+	hint: 'See all supported types: https://github.com/rich-harris/devalue',
```

Comment:
Slight personal preference towards the old, because I do like pointing out that this is an external (not Astro) link. I like telling people where they are going, and most of our error messages take us to our own documentation.